gatsby-node.js

```diff
@@ -236,15 +252,17 @@ exports.createPages = async ({ graphql, actions }) => {
         })
       })
     })
-  })
-  result.data.allMeasuresJson.edges.forEach(({ node }) => {
-    createPage({
-      path: `/${node.electionDate}/referendum/${node.fields.slug}`,
-      component: path.resolve("src/templates/measure.js"),
-      context: {
-        id: node.id,
-        slug: node.fields.slug,
-      },
+    node.Referendums.forEach(referendum => {
+      createPage({
+        path:
+          "/" +
```

Comment:
A template string might be a little easier to read here

Reply:
Fixed in new commit